msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/DiscussionForumTool.java

```diff
@@ -1893,56 +1941,75 @@ private String processTopicSettings(boolean draft){
 
   private String saveTopicSettings(boolean draft)
   {
-  	log.debug("saveTopicSettings(" + draft + ")");
-  	setPermissionMode(PERMISSION_MODE_TOPIC);
+    log.debug("saveTopicSettings(" + draft + ")");
```

Comment:
if you modify a logging statement, should correct the format to use:

```suggestion
    log.debug("saveTopicSettings({})", draft);
```

Reply:
Changed!